akka-bbb-apps/project/Dependencies.scala

```diff
@@ -60,15 +60,15 @@ object Dependencies {
 
   object Test {
     val scalaTest = "org.scalatest" %% "scalatest" % Versions.scalaTest % "test"
-    val junit = "junit" % "junit" % Versions.junit % "test"
+//    val junit = "junit" % "junit" % Versions.junit % "test"
     val mockitoCore = "org.mockito" % "mockito-core" % Versions.mockito % "test"
     val scalactic = "org.scalactic" % "scalactic_2.12" % Versions.scalactic % "test"
     val akkaTestKit = "com.typesafe.akka" %% "akka-testkit" % Versions.akkaTestKit % "test"
   }
 
   val testing = Seq(
     Test.scalaTest,
-    Test.junit,
+//    Test.junit,

```

Comment:
@paultrudel are you able to run the tests if you uncomment the 2 lines in this file?

Reply:
In direct conversation @paultrudel confirmed the tests are fine. Seems these two lines are commented out because they are not used.